src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix4x4.Impl.cs

```diff
@@ -232,7 +232,38 @@ public static Impl CreateBillboard(in Vector3 objectPosition, in Vector3 cameraP
 
                 result.X = axisX.AsVector4();
                 result.Y = axisY.AsVector4();
-                result.Z = axisZ.AsVector4();;
+                result.Z = axisZ.AsVector4();
+                result.W = Vector4.Create(objectPosition, 1);
+
+                return result;
+            }
+
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Please add XML comments to the new public APIs. I realize the existing ones don't have docs, but eventually we want to end up in a place where the XML docs are the source of truth, and in the meantime we use XML docs on new methods to seed the documentation, automatically exporting it as part of the release.

Reply:
Nevermind, I see that these are on an internal type.